app/assets/stylesheets/browser.scss

```diff
@@ -25,7 +25,7 @@ $selected-sample-outline-color: #e0ffff;
   }
 
   .page-selected {
-    background: #ff0;
+    background: #d2f8d2;
```

Comment:
Does this correspond to a color variable that you could use here?  If not, please make one.